client/dashboard/sites/overview/plan-card.tsx

```diff
@@ -26,9 +55,33 @@ export default function PlanCard( { site }: { site: Site } ) {
 			heading={ site.plan?.product_name_short || '\u00A0' }
 			description={ getCardDescription( plan, purchase ) }
 			tracksId="plan"
-			variant={ isLoadingPlan || isLoadingPurchase ? 'loading' : undefined }
+			variant={
+				isLoadingPlan || isLoadingPurchase || isLoadingMediaStorage ? 'loading' : undefined
+			}
 			link={ site.plan?.is_free ? undefined : '/v2/me/billing/active-subscriptions' }
-			bottom={ <div /> }
+			bottom={
+				<VStack spacing={ 4 }>
+					<Stat
+						density="high"
+						strapline={ __( 'Storage' ) }
+						metric={ filesize( mediaStorage?.storage_used_bytes ?? 0, { round: 0 } ) }
+						description={ filesize( mediaStorage?.max_storage_bytes ?? 0, { round: 0 } ) }
+						descriptionAlignment="end"
```

Comment:
These descriptions in the plans card are right aligned

<img width="632" height="276" alt="CleanShot 2025-07-23 at 19 02 59@2x" src="https://github.com/user-attachments/assets/2ae5d807-d3b4-475a-8887-c62e674d250e" />

However all the examples from the `Stats` figma show left aligned

<img width="636" height="272" alt="CleanShot 2025-07-23 at 19 04 36@2x" src="https://github.com/user-attachments/assets/9fc21dd5-6626-473f-ad2a-13ac7fcddb38" />

@matt-west are you happy with this inconsistency with the reference component? Maybe we should use "of 200 GB" instead. So it can be the same as all other stats.

Reply:
For sure, but I mean, couldn't we do that for every metric?
Why shouldn't all the other examples in the DS figma be redone so they represent the end of the progress bar.
For example, if 1.3 M is "33% left", why shouldn't that be right aligned and changed to "2 M"?

Reply:
Sounds good, I'm baking in that logic here https://github.com/Automattic/wp-calypso/pull/104940

Looking again at the example stats in Figma, the ones that descriptions on the left do look like they wouldn't have progress bars in real life.